herddb-core/src/main/java/herddb/server/ServerConfiguration.java

```diff
@@ -68,7 +69,7 @@
 
     public final static String PROPERTY_DEFERRED_SYNC_PERIOD = "txlog.deferredsyncperiod";
     public final static int PROPERTY_DEFERRED_SYNC_PERIOD_DEFAULT = 0;  /* disabled */
-    
+  
     public final static String PROPERTY_TXLOG_USE_ODIRECT = "txlog.use_o_direct";
     public final static boolean PROPERTY_TXLOG_USE_ODIRECT_DEFAULT = false;
```

Comment:
Can we add a system property for this default and the others?
Just a 'herddb.file.use_o_direct_default'
To be used for all of the 3 params.

I would like to setup a CI job which runs all the suite with o_direct=true

This way we can cover better all of the usages

Reply:
Added a system property herddb.file.use_o_direct_default to handle all 3 defaults